tools/topology/sof-tgl-max98357a-rt5682.m4

```diff
@@ -84,7 +84,7 @@ define(DMIC_PIPELINE_16k_ID, `11')
 define(DMIC_PIPELINE_KWD_ID, `12')
 define(DMIC_DAI_LINK_16k_ID, `2')
 # define pcm, pipeline and dai id
-define(KWD_PIPE_SCH_DEADLINE_US, 20000)
+define(KWD_PIPE_SCH_DEADLINE_US, 5000)
```

Comment:
why does this improve performance and what does 'performance' means here.

Reply:
@plbossart here by *performace* improvement I mean how much work we can do in 1 [ms] time period. This patch improves performance because we process less samples (4 times less) and therefore lowered CPU load.

Reply:
but dont we still have to process the full draining buffer ? This just looks like we are splitting into smaller chunks, bust should EDF be pre-emptive anyway so chunk size should not matter here ?

Reply:
@lgirdwood no, this has nothing to do with draining/history buffer. This is about the real time steam - when its size is 20 [ms] then you need more MIPS to process it than when the period is only 5 ms.